Error connecting nebula with console on M1

Nebula Graph version 2.5.0
Apple M1 chip
When I use docker to connect to the console, the following problems occur.
```
foesa@foesasdeMacBook-Pro nebula-docker-compose % docker run --rm -ti --network nebula-docker-compose_nebula-net --entrypoint=/bin/sh vesoft/nebula-console:v2.5.0
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

/ # nebula-console -u root -p 23 --address=graphd --port=9669
2021/09/10 01:47:47 [INFO] connection pool is initialized successfully
2021/09/10 01:47:47 Fail to create a new session from connection pool, failed to open transport, error: dial tcp 172.21.0.9:9669: connect: connection refused
panic: Fail to create a new session from connection pool, failed to open transport, error: dial tcp 172.21.0.9:9669: connect: connection refused

goroutine 1 [running]:
log.Panicf(0x7ac363, 0x35, 0xc000061e98, 0x1, 0x1)
	/usr/local/go/src/log/log.go:345 +0xc0
main.main()
	/usr/src/main.go:364 +0x8cd
```
And the state is `unhealthy`
```
foesa@foesasdeMacBook-Pro nebula-docker-compose % docker-compose ps
              Name                             Command                   State                                                          Ports                                                  
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
nebula-docker-compose_graphd1_1     /usr/local/nebula/bin/nebu ...   Up (healthy)     0.0.0.0:61914->19669/tcp, 0.0.0.0:61912->19670/tcp, 0.0.0.0:61913->9669/tcp                              
nebula-docker-compose_graphd2_1     /usr/local/nebula/bin/nebu ...   Up (healthy)     0.0.0.0:61919->19669/tcp, 0.0.0.0:61917->19670/tcp, 0.0.0.0:61918->9669/tcp                              
nebula-docker-compose_graphd_1      /usr/local/nebula/bin/nebu ...   Up (healthy)     0.0.0.0:61915->19669/tcp, 0.0.0.0:61916->19670/tcp, 0.0.0.0:9669->9669/tcp                               
nebula-docker-compose_metad0_1      /usr/local/nebula/bin/nebu ...   Up (unhealthy)   0.0.0.0:61901->19559/tcp, 0.0.0.0:61899->19560/tcp, 0.0.0.0:61900->9559/tcp, 9560/tcp                    
nebula-docker-compose_metad1_1      /usr/local/nebula/bin/nebu ...   Up (unhealthy)   0.0.0.0:61892->19559/tcp, 0.0.0.0:61893->19560/tcp, 0.0.0.0:61894->9559/tcp, 9560/tcp                    
nebula-docker-compose_metad2_1      /usr/local/nebula/bin/nebu ...   Up (unhealthy)   0.0.0.0:61896->19559/tcp, 0.0.0.0:61897->19560/tcp, 0.0.0.0:61898->9559/tcp, 9560/tcp                    
nebula-docker-compose_storaged0_1   /usr/local/nebula/bin/nebu ...   Up (unhealthy)   0.0.0.0:61902->19779/tcp, 0.0.0.0:61903->19780/tcp, 9777/tcp, 9778/tcp, 0.0.0.0:61904->9779/tcp, 9780/tcp
nebula-docker-compose_storaged1_1   /usr/local/nebula/bin/nebu ...   Up (unhealthy)   0.0.0.0:61910->19779/tcp, 0.0.0.0:61911->19780/tcp, 9777/tcp, 9778/tcp, 0.0.0.0:61909->9779/tcp, 9780/tcp
nebula-docker-compose_storaged2_1   /usr/local/nebula/bin/nebu ...   Up (unhealthy)   0.0.0.0:61906->19779/tcp, 0.0.0.0:61907->19780/tcp, 9777/tcp, 9778/tcp, 0.0.0.0:61905->9779/tcp, 9780/tcp
```

close #2791

--- a/.github/workflows/nightly.yml
+++ b/.github/workflows/nightly.yml
@@ -62,17 +62,21 @@ jobs:
           - tools
     steps:
       - uses: actions/checkout@v2
-      - name: Build and push Docker images
-        uses: docker/build-push-action@v1
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
+      - uses: docker/login-action@v1
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: vesoft/nebula-${{ matrix.service }}
-          path: .
-          dockerfile: docker/Dockerfile.${{ matrix.service }}
-          tags: nightly,v2-nightly
-          cache_froms: vesoft/nebula-dev:centos7
-          add_git_labels: true
+      - uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./docker/Dockerfile.${{ matrix.service }}
+          platforms: linux/amd64,linux/arm64
+          tags: |
+            vesoft/nebula-${{ matrix.service }}:nightly
+            vesoft/nebula-${{ matrix.service }}:v2-nightly
+          push: true
 
   coverage:
     name: coverage
@@ -113,12 +117,12 @@ jobs:
               -DENABLE_TESTING=on \
               -DENABLE_COVERAGE=on \
               -B build
-          echo "::set-output name=j::10"
+          echo "::set-output name=j::8"
           echo "::set-output name=t::6"
       - name: Make
         run: |
           ccache -z
-          cmake --build build/ -j $(nproc) --target nebula-metad nebula-storaged nebula-graphd
+          cmake --build build/ -j ${{ steps.cmake.outputs.j }} --target nebula-metad nebula-storaged nebula-graphd
           cmake --build build/ -j ${{ steps.cmake.outputs.t }}
           ccache -s
       - name: CTest

--- a/.github/workflows/nightly.yml
+++ b/.github/workflows/nightly.yml
@@ -75,7 +75,6 @@ jobs:
           platforms: linux/amd64,linux/arm64
           tags: |
             vesoft/nebula-${{ matrix.service }}:nightly
-            vesoft/nebula-${{ matrix.service }}:v2-nightly
           push: true
 
   coverage:
@@ -88,7 +87,6 @@ jobs:
           - ubuntu2004
         compiler:
           - gcc-9.2
-
     container:
       image: vesoft/nebula-dev:${{ matrix.os }}
       env:

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -78,6 +78,7 @@ jobs:
             tag="vesoft/nebula-${{ matrix.service }}:latest"
           fi
           echo "::set-output name=tag::$tag"
+      - uses: docker/setup-qemu-action@v1
       - uses: docker/setup-buildx-action@v1
       - uses: docker/login-action@v1
         with:
@@ -87,6 +88,7 @@ jobs:
         with:
           context: .
           file: ./docker/Dockerfile.${{ matrix.service }}
+          platforms: linux/amd64,linux/arm64
           tags: |
             vesoft/nebula-${{ matrix.service }}:${{ steps.tagname.outputs.tag }}
             vesoft/nebula-${{ matrix.service }}:${{ steps.tagname.outputs.majorver }}

